README.md

```diff
@@ -1,158 +1,111 @@
+Deploying the CDI controller is straight forward.  Create the controller in the namespace where VM images are to be stored.  Here, `default` is used.  In a production setup, a namespace that is inaccessible to regular users should be used instead.
 
-**Object Store:** Arbitrary url-based storage location.  Currently we support http and S3 protocols.
+`$ kubectl -n default create -f https://raw.githubusercontent.com/kubevirt/containerized-data-importer/master/manifests/cdi-controller-deployment.yaml`
 
-**Storage Class:** Long-lived, default Storage Class which links Persistent Volume Claims to the desired Dynamic Provisioner(s). Referenced by the golden PVC. The example makes use of the "default" provisioner; however, any provisioner that manages mountable volumes is compatible.
+To begin importing images, follow the PVC configuration guide below.
 
-## Running the Data Importer
+### Start Importing Images
 
-Deploying the controller and importer pod fairly simple but requires several manual steps. An admin is expected to deploy the following object using `kubectl`. If the admin's current context is outside of the "golden" namespace then she is expected to use the `--namespace=<name>` kubectl flag:
-1. "golden" namespace where the controller, importer, secret(s) and PVC(s) live.
-1. secret(s) containing endpoint credentials. Not required if the endpoint(s) are public.
-1. storage class(es) defining the backend storage provisioner(s).
-1. controller pod via the Deployment template. Note that the controller pod spec needs to set an environment variable named OWN_NAMESPACE which can be done as:
-```
-...
-   imagePullPolicy: Always
-       env:
-         - name: OWN_NAMESPACE
-           valueFrom:
-             fieldRef:
-               fieldPath: metadata.namespace
-```
+> Note: The CDI controller is a require part of this work flow.
 
-### Assumptions
-- A running Kubernetes cluster
-- A reachable object store
-- A file in the object store to be imported.
-
-### Configuration
-
-Make copies of the [example manifests](./manifests/importer) to some local directory for editing.  There are several values required by the data importer pod that are provided by the configMap and secret.
-
-The files needed are:
-- cdi-controller-pod.yaml
-- endpoint-secret.yaml
+Make copies of the [example manifests](./manifests/example) for editing. The neccessary files are:
 - golden-pvc.yaml
+- endpoint-secret.yaml
 
-#### cdi-controller-pod.yaml
-(to be replaced by a Deployment manifest)
+###### Edit golden-pvc.yaml:
+1.  `storageClassName:` The default StorageClass will be used if not set.  Otherwise, set to a desired StorageClass
```

Comment:
How about: _Add `StorageClassName: <target-storage-class>` under the `spec:` stanza where the target storage class is suitable for high-speed image cloning. If the storage class is omitted then a default storage class will be used._ ?

Reply:
This restriction would be only imposed in documentation though.  CDI doesn't actually necessitate clonable storage.  

Also, there could be a use case where CDI isn't coupled with CNV at all and is used to generate ad-hoc PVCs for use by Kubevirt.